### PR TITLE
Make Log Search and Prometheus images configurable

### PR DESCRIPTION
**Pull request by dvaldivia**
Three new fields were added to let users customize the docker images for log search and prometheus

Signed-off-by: Daniel Valdivia <hola@danielvaldivia.com>

Screen Shot 2021-05-04 at 9 19 07 AM

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx [hunk 1]
```diff
@@ -151,19 +157,43 @@ const Configure = ({
...
+        {
+          fieldKey: "logSearchPostgresImage",
+          required: false,
+          value: logSearchPostgresImage,
+          pattern: /^((.*?)\/(.*?):(.+))$/,
+          customPatternMessage:
+            "Format must be of form: 'library/postgres:VERSION'",
+        },
+        {
+          fieldKey: "prometheusImage",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:190]: Validation is not being triggered when we change the field, it only triggers when you modify other input. please add the dependency to useEffect

Screen Shot 2021-05-04 at 12 16 23
Screen Shot 2021-05-04 at 12 18 15

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx [hunk 2]
```diff
@@ -196,6 +226,7 @@ const Configure = ({
     customImage,
     imageName,
     consoleImage,
+    logSearchImage,
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:229]: Need to add logSearch & prometheus Image vars here:

``` Line 225:6:  React Hook useEffect has missing dependencies: 'logSearchPostgresImage' and 'prometheusImage'. Either include them or remove the dependency array  react-hooks/exhaustive-deps```

File: portal-ui/src/screens/Console/Tenants/AddTenant/AddTenant.tsx [hunk 3]
```diff
@@ -47,6 +47,7 @@ import Encryption from "./Steps/Encryption";
 import TenantSize from "./Steps/TenantSize";
 import Preview from "./Steps/Preview";
 import Affinity from "./Steps/Affinity";
+import { log } from "util";
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/AddTenant/AddTenant.tsx:50]: Non used var, we can remove

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Tenants/AddTenant/AddTenant.tsx (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/AddTenant/AddTenant.tsx:50]: done

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:190]: done

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:229]: done

**Review by bexsoft — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]
LGTM

**Pull request merged**